script/generate-travis-config.ts

```diff
@@ -13,6 +13,16 @@ function getLFSVersion() {
 }
 
 function getConfig(platform: string, arch: string) {
+  // if this is macOS, we don't need
+  // to read anything from `dependencies`
+  if (platform === 'darwin') {
```

Comment:
I noticed we dropped the `arch === 'amd64'` check and wanted to double check that we're not losing anything important by doing so (see https://github.com/desktop/dugite-native/pull/320/files#diff-775ff3619560ef0cd5cdf7b0704d1f5aL68)

Reply:
i think this is ok since we need to build macOS regardless of architecture. i'm also pretty sure we don't support other architectures for macOS

Reply:
Thanks for verifying!